### ISSUE TITLE
Lesson 23, stacks example, array items doesn't match gems.

### ISSUE THREAD

**Issue by duianto** [GDQuest/learn-gdscript#406]
**Issue description:**

Lesson 23, stacks example, array items doesn't match gems in the left panel.

**To Reproduce**
Steps to reproduce the bug:
1. Go to https://gdquest.github.io/learn-gdscript/staging/#course/lesson-23-append-to-arrays/lesson.tres
2. Scroll down to the heading: `Using arrays as stacks`
3. See the first interactive example.

**Observed behavior**

The left panel shows the following gems, from top to bottom:
- blue gem
- yellow gem
- fire gem

But the code in the right panel uses the same name for all three items in the array:
```
var crates = ["fire gem", "fire gem", "fire gem"]
```

**Expected behavior**

The code should probably match the gems in the left panel.
```
var crates = ["fire gem", "yellow gem", "blue gem"]
```

**Screenshots**

![msedge_DatlRqXy3I](https://user-images.githubusercontent.com/13420573/156647354-5a458880-3b93-4c98-9c1e-e83fd4265b7b.png)

**Information about your device (please complete the following information):**

 - Operating System: Windows 10 21H1
 - Browser: Microsoft Edge 98.0.1108.62 (Official build) (64-bit)

**Comment by duianto** [comment]
The gems in the left panel seem to change.
A `green heart` gem appeared at the top this time:

![msedge_svGDCXg9rh](https://user-images.githubusercontent.com/13420573/156722105-12777db9-8eda-4b7e-9f96-9b06aa4cb1ef.png)

**Comment by johnnygossdev** [comment]
Fixed via e0ea530f0d92459ea1faeb84d40e6a082c6c8c8c
